libkbfs/backpressure_disk_limiter_test.go

```diff
@@ -487,13 +487,13 @@ func testBackpressureDiskLimiterSmallDiskDelay(
 		// When called in subsequent times from
 		// beforeBlockPut, simulate the journal taking up
 		// space.
-		return diskBytes - bdl.byteTracker.used,
-			diskFiles - bdl.fileTracker.used, nil
+		return diskBytes - bdl.journalByteTracker.used,
+			diskFiles - bdl.journalFileTracker.used, nil
 	}
 
 	log := logger.NewTestLogger(t)
 	bdl, err := newBackpressureDiskLimiterWithFunctions(
-		log, 0.1, 0.9, 0.25, math.MaxInt64, math.MaxInt64,
+		log, 0.1, 0.9, 0.25, 0.1, math.MaxInt64, math.MaxInt64,
```

Comment:
Doesn't have to be as comprehensive as the existing tests, but it would be nice to have at least one test for the interaction between the cache and journal limits.

Could be something like the scenario I described in the previous review:

1) Set up journal and cache with 25% each of 100 GB.
2) Fill them both up.
3) Test that the journal isn't in backpressure, and that the cache doesn't need to evict.